docs/en/reference/second-level-cache.rst

```diff
@@ -79,23 +79,17 @@ Cache Regions
 
 ``Doctrine\ORM\Cache\Region\DefaultRegion`` It's the default implementation.
  A simplest cache region compatible with all doctrine-cache drivers but does not support locking.
-
-``Doctrine\ORM\Cache\Region`` and ``Doctrine\ORM\Cache\ConcurrentRegion``
-Defines contracts that should be implemented by a cache provider.
+``Doctrine\ORM\Cache\Region`` and ``Doctrine\ORM\Cache\ConcurrentRegion`` define contracts that should be implemented by a cache provider.
 
 It allows you to provide your own cache implementation that might take advantage of specific cache driver.
 
-If you want to support locking for ``READ_WRITE`` strategies you should implement ``ConcurrentRegion``; ``CacheRegion`` otherwise.
+If you want to support locking for ``READ_WRITE`` strategies you should implement ``ConcurrentRegion`` (``CacheRegion`` otherwise).
```

Comment:
Why change this? the second part is not a precision, it's the counterpart of the first one, and has a similar importance.

Reply:
I reverted this change